private-api/app/src/VotingIrregularities.Api/Properties/launchSettings.json

```diff
@@ -11,7 +11,7 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
-      "launchUrl": "swagger/ui",
+      "launchUrl": "swagger",
```

Comment:
should be <http://localhost:5000/swagger> instead of <http://localhost:5000/swagger/ui> in readme ?